### PR TITLE
Give up on animated flags for now. Implement vector flag with tint.

### PR DESCRIPTION
**Pull request by wolfd**

### CODE REVIEW

File: app/src/main/res/layout/activity_flag_creation.xml [hunk 1]
```diff
@@ -10,13 +11,22 @@
...
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        android:id="@+id/flag_color" />
+    <ImageButton
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:srcCompat="@drawable/ic_flag"
+        android:id="@+id/flag_button"
+        android:scaleType="fitCenter"
```
Comment (segerphilip) [inline app/src/main/res/layout/activity_flag_creation.xml:19]: We'll probably need to play with sizing, but this works fine for now

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -2,5 +2,5 @@
 <resources>
     <color name="colorPrimary">#3F51B5</color>
     <color name="colorPrimaryDark">#303F9F</color>
-    <color name="colorAccent">#FF4081</color>
+    <color name="colorAccent">#ff4081</color>
```
Comment (segerphilip) [inline app/src/main/res/values/colors.xml:5]: Um, maybe fix this?

**Review by segerphilip — APPROVED** [review]
Looks good and tested, besides one possible style change :shipit:

File: app/src/main/res/values/colors.xml (hunk 2 above)
Comment (wolfd) [reply app/src/main/res/values/colors.xml:5]: This just gets automatically changed by intelliJ for some reason.

Comment (segerphilip) [reply app/src/main/res/values/colors.xml:5]: Huh. Okay, sounds good.

**Pull request merged**